src/directives/modal/modal.js

```diff
@@ -47,7 +47,7 @@ const bind = (el, binding, vnode) => {
       if (!isDisabled(currentTarget)) {
         const type = evt.type
         // Open modal only if trigger is not disabled
-        if (type === 'click' || (type === 'keydown' && evt.keyCode === 32)) {
+        if (type === 'click' || (type === 'keydown' && (evt.keyCode === 13 || evt.keyCode === 32))) {
```

Comment:
We should import the KeyCode object from `./../utils/key-kodes` rather than using literal numeric values.

```js
import KeyCodes from '../../utils/key-codes'

// ...

const keyCode = evt.keyCode
if (type === 'click' || (type === 'keydown' && (keyCode === KeyCodes.SPACE || keyCode === KeyCodes.ENTER))) 

```

Reply:
Tweaked the code a bit to use the key-codes utility and linted